content/docs/services/s3.md

```diff
@@ -4,7 +4,7 @@ menu:
     parent: services
 title: S3
 name: "s3"
-description: "Amazon Simple Storage System (persistent storage)"
+description: "Amazon S3 provides developers with secure, durable, highly-scalable object storage"
```

Comment:
Hm, this one is a sentence but the others are sentence fragments - can we make this one make the RDS one better?